src/rpc/core_rpc_server.cpp

```diff
@@ -126,14 +152,29 @@ namespace cryptonote
   bool core_rpc_server::on_get_height(const COMMAND_RPC_GET_HEIGHT::request& req, COMMAND_RPC_GET_HEIGHT::response& res)
   {
     PERF_TIMER(on_get_height);
+    bool r;
+    if (use_bootstrap_daemon_if_necessary<COMMAND_RPC_GET_HEIGHT>("JON", "/getheight", req, res, r))
```

Comment:
Typo here? `JON` -> `JSON`? (applies to all other uses of `use_bootstrap_daemon_if_necessary` as well)

Reply:
It doesn't really matter, but it's intentional, not a typo. They come from macro names `MAP_URI_AUTO_JON2, MAP_URI_AUTO_BIN2, MAP_JON_RPC_WE` used in https://github.com/monero-project/monero/blob/master/src/rpc/core_rpc_server.h.